rlp/src/stream.rs

```diff
@@ -326,7 +325,7 @@ impl RlpStream {
 		BasicEncoder::new(self)
 	}
 
-	/// Finalize current ubnbound list. Panics if no unbounded list has been opened.
+	/// Finalize current unbounded list. Panics if no unbounded list has been opened.
 	pub fn complete_unbounded_list(&mut self) {
```

Comment:
Unrelated, but maybe this should be called `finalize_unbounded_list()`?

Reply:
Both sounds fine to me, but I'm not a native speaker. Also, I'd like to avoid breaking changes.

Reply:
"complete" can be both the imperative form of the verb as in "complete your homework before dinner!" or be and adjective "a complete list of my belongings". 
"finalize" does not have this ambiguity.

In this context the ambiguity makes it unclear if the method returns an unbounded list of something that is complete, or that it takes an unbounded list and completes it.

We can add a copy named `finalize_unbounded_list` and deprecate this one. In another PR.

Reply:
opened #204